games-arcade/goonies/goonies-1.4.1528.recipe

```diff
@@ -15,39 +15,36 @@ treasure and escape back to save the Goon Docks from demolition?"
 HOMEPAGE="http://www2.braingames.getput.com/goonies/"
 COPYRIGHT="2006-2009 Brain Games"
 LICENSE="GNU GPL v2"
-REVISION="3"
+REVISION="4"
 SOURCE_URI="http://braingames.jorito.net/goonies/downloads/goonies.src_1.4.1528.tgz"
-CHECKSUM_SHA256="9e9c1e67a41dfae512e975b6931828cdc82f07397921ca0a41a81c3020a73799"
+CHECKSUM_SHA256="09e4a022ea4c31fe0f01393ba82ffaf44dbfbf0edcc2da9f569d9fe061382ce6"
 PATCHES="goonies-$portVersion.patchset"
 
-ARCHITECTURES="!x86_gcc2 x86"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	goonies$secondaryArchSuffix = $portVersion
-	app:goonies
+	app:Goonies
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
 	lib:libGLU$secondaryArchSuffix
-	lib:libsdl$secondaryArchSuffix
-	lib:libSDL_image$secondaryArchSuffix
-	lib:libSDL_mixer$secondaryArchSuffix
-#	lib:libSDL_sound$secondaryArchSuffix
-#	lib:libSDL_ttf$secondaryArchSuffix
+	lib:libSDL$secondaryArchSuffix
```

Comment:
Shouldn't it be libSDL_1.2?

Reply:
Seems I missed this one, thanks for checking, will look in a bit :)

Reply:
Should be ok now :)